### PR TITLE
new profile info section

### PR DESCRIPTION
**Pull request by tarafanlin**
new profile info section and some nit styling fixes for external slate & profile pages

Screen Shot 2020-10-29 at 12 27 13 PM

### CODE REVIEW

**Comment by martinalong** [comment]
Do you think you can fix the alignment so that it also looks the same on the internal view? It looks centered here 
image

**Comment by martinalong** [comment]
Also if you are on your own profile, it shouldn't show the follow button. I think the old follow button I had has that logic. You can look at that to see how it did that.

**Comment by martinalong** [comment]
I'm guessing haris will also have some nits to make on alignment / spacing. So maybe send him a screenshot to get his opinion? Generally it's a good idea to do that when you make a change so that we can keep teh design consistent throughout Slate

**Comment by jimmylee** [comment]
Love the initiative, would be nice to get @harisbutt involved with some of the review here.

**Comment by tarafanlin** [comment]
added some fixes for the internal profile view here. preserving the buttons & functionality we have.
Screen Shot 2020-10-29 at 4 38 48 PM

**Comment by martinalong** [comment]
Was adding this top bar here intentional? If not can you remove it from this view? Let me know if you need help doing that
image

File: components/core/Profile.js [hunk 1]
```diff
@@ -8,40 +8,125 @@ import { ProcessedText } from "~/components/system/components/Typography";
...
   width: 100%;
-  margin-top: 16px;
+  padding: 24px 24px 0px 24px;
+`;
+
+const STYLES_PROFILE_INFO = css`
+  display: flex;
+  line-height: 1.3;
+  margin: 0 auto;
+  width: 50%;
```
Comment (martinalong) [inline components/core/Profile.js:19]: Can you make this:
``` 
width: 100%;
max-width: ${Constants.sizes.desktop}px;
```
That's the width and max width we have on the slate blocks that make up teh rest of the profile page

**Comment by tarafanlin** [comment]
> Was adding this top bar here intentional? If not can you remove it from this view? Let me know if you need help doing that
> image

hmmm I missed that. will patch it

File: components/core/Profile.js (hunk 1 above)
Comment (tarafanlin) [reply components/core/Profile.js:19]: cool! will do

**Comment by martinalong** [comment]
And this is a weird edge case that happens on the external view.Not sure what's causing it
image

**Comment by tarafanlin** [comment]
> And this is a weird edge case that happens on the external view when you have a really long one-word name.Not sure what's causing it to happen on external but not internal view
> image

hmmm interesting. I will take a look to see if I can reproduce this bug with long words

**Comment by jimmylee** [comment]
You need `overflow-wrap: break-word;` @tarafanlin

**Comment by tarafanlin** [comment]
> You need `overflow-wrap: break-word;` @tarafanlin

Went into a rabbit hole debugging this. 
What do you mean by "won't work if you have weird dom inline element nesting"? I tried `overflow-wrap` and it still has the issue.
Like @martinalong mentioned, I'm not sure why it works for the internal view but not an external view. Did I miss something? The styling I have is basically the same for those two in the inspector.
Screen Shot 2020-10-30 at 12 35 20 AM

Screen Shot 2020-10-30 at 12 33 37 AM

**Comment by jimmylee** [comment]
Sometimes it depends on the parent element.

**Comment by jimmylee** [comment]
I'll have a look tomorrow morning.

**Comment by tarafanlin** [comment]
> I'll have a look tomorrow morning.

thanks cake!🙏

**Comment by tarafanlin** [comment]
> Sometimes it depends on the parent element.

you're right it's the parent element. it's fixed!

**Comment by martinalong** [comment]
Looks like it's fixed! 

I'm guessing haris will want you to align this follow button vertically with the name or move it to below the description like how you have it on the internal version. Might be good to do one last quick check with him 
image
Like this
image

**Comment by martinalong** [comment]
And it looks like the white bar is still on the login page. Do you need help removing it?
image

**Comment by tarafanlin** [comment]
> Looks like it's fixed!
> 
> I'm guessing haris will want you to align this follow button vertically with the name or move it to below the description like how you have it on the internal version. Might be good to do one last quick check with him
> image
> Like this
> image

ok will do!

**Comment by tarafanlin** [comment]
> > I'm guessing haris will want you to align this follow button vertically with the name or move it to below the description like how you have it on the internal version. Might be good to do one last quick check with him

fixes 
Screen Shot 2020-10-31 at 12 57 29 PM
Screen Shot 2020-10-31 at 12 57 35 PM
Screen Shot 2020-10-31 at 12 59 21 PM

**Comment by tarafanlin** [comment]
there is some issue with this branch so I created a new PR.

**Pull request closed without merging**